modules/apps/content-dashboard/content-dashboard-web/test/js/components/Sidebar.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {act, cleanup, render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React, {useState} from 'react';
+
+import Sidebar from '../../../src/main/resources/META-INF/resources/js/components/Sidebar';
+
+import '@testing-library/jest-dom/extend-expect';
+
+const ControlledSidebar = ({open, subtitle = '', title = ''}) => {
+	const [isOpen, setIsOpen] = useState(open);
+
+	return (
+		<Sidebar onClose={() => setIsOpen(false)} open={isOpen}>
+			<Sidebar.Header subtitle={subtitle} title={title} />
+		</Sidebar>
+	);
+};
+
+describe('Sidebar', () => {
+	beforeEach(() => {
+		jest.useFakeTimers();
+	});
+
+	afterEach(() => {
+		jest.clearAllTimers();
+		cleanup();
+	});
+
+	it('renders an open sidebar', () => {
+		const {asFragment} = render(<Sidebar />);
+
+		act(() => {
+			jest.runAllTimers();
+		});
```

Comment:
How come you need to runAllTimers in each of these tests? By using `asFragment`, I would think it wouldn't be necessary unless there is an explicit interaction or delay you are accounting for

Reply:
There is a delay when you render the Sidebar as open for the very first time, without that delay the animation was not happening, so we need to run the timers here.

Reply:
Ah gotcha!